aws/data_source_aws_ec2_local_gateway_virtual_interface_group.go

```diff
@@ -0,0 +1,96 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+)
+
+func dataSourceAwsEc2LocalGatewayVirtualInterfaceGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsEc2LocalGatewayVirtualInterfaceGroupRead,
+
+		Schema: map[string]*schema.Schema{
+			"filter": ec2CustomFiltersSchema(),
+			"id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"local_gateway_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"local_gateway_virtual_interface_ids": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"tags": tagsSchema(),
```

Comment:
`tags` can be provided as an argument for filtering: https://github.com/terraform-providers/terraform-provider-aws/pull/13767/files#diff-e466993d721ef8f751f7e6fd9172444cR54-R56

Reply:
Ah yeah, you are correct -- I'll try to add covering tests as well using the new `aws_ec2_tag` resource (since this resource implicitly exists and cannot be managed). 😄